torch/_torch_docs.py

```diff
@@ -2698,6 +2707,8 @@
 
 If :attr:`mat` is a `n x m` Tensor, :attr:`vec` is a 1D Tensor of size `m`, :attr:`out` will be 1D of size `n`.
 
+.. note:: This function does not :ref:`broadcast <broadcasting-semantics>`.
```

Comment:
Maybe one benefit of using the low-level functions is that the dimensionality of the tensors are explicit?